packages/lesswrong/server/repos/NotificationsRepo.ts

```diff
@@ -0,0 +1,171 @@
+import AbstractRepo from "./AbstractRepo";
+import Notifications from "../../lib/collections/notifications/collection";
+import { READ_WORDS_PER_MINUTE } from "../../lib/collections/posts/schema";
+import { getSocialPreviewSql } from "../../lib/collections/posts/helpers";
+import type { NotificationDisplay } from "../../lib/notificationTypes";
+
+// This should return an object of type `NotificationDisplayUser`
+const buildNotificationUser = (prefix: string) =>
```

Comment:
This is basically a manual version of what the `sql-fragments` branch does. We can probably rewrite this in a simpler and less outside-the-type-system way once that branch gets merged, but I don't thinks it's worth delaying this to wait for that.